workshops/templates/workshops/problems.html

```diff
@@ -1,21 +1,57 @@
 {% extends "workshops/_page.html" %}
 
 {% block content %}
-<h3>Workshops Missing Attendance Figures</h3>
-{% if missing_attendance %}
-<ul>
-{% for event in missing_attendance %}
-<li>
-  {% if event.mailto %}
-    <a href="mailto:{% for m in event.mailto %}{{ m }}{% if not forloop.last %},{% endif %}{% endfor %}?subject={{subject | urlencode}}{{event}}&body={{body_pre | urlencode}}{{event}}{{body_post | urlencode}}">
-  {% endif %}
-    {{ event.get_ident }}
-  {% if event.mailto %}
-    </a>
-  {% endif %}
-</li>
-{% endfor %}
-</ul>
+{% if events %}
+<p><span class="glyphicon glyphicon-envelope"></span> = mail instructors and host about problem</p>
+<p><span class="glyphicon glyphicon-remove"></span> = no email addresses available</p>
+<table class="table table-striped">
+  <tr>
+    <th>Event</th>
+    <th>Attendance?</th>
+    <th>Location?</th>
+    <th>Dates?</th>
+  </tr>
+  {% for event in events %}
+  <tr>
+    <td>
+      <a href="{% url 'event_details' event.get_ident %}">{{ event }}</a>
+    </td>
+    <td>
+      {% if event.missing_attendance_ %}
+        {% if event.mailto_ %}
+          <a href="mailto:{{e.mailto_}}?subject={{'missing attendance figures for ' | urlencode}}{{event.get_ident}}&body={{'Hi, Can you please send us attendance figures for ' | urlencode}}{{event.get_ident}}{{'?  Thanks for your help.' | urlencode}}">
```

Comment:
I know the solution for multiline text filtered with `urlencode`. It's https://docs.djangoproject.com/en/1.8/ref/templates/builtins/#filter - care to upgrade your templates?